code/src/cljs/sixsq/nuvla/ui/edges_detail/views.cljs

```diff
@@ -1871,7 +1872,11 @@
        (events-plugin/events-section
          {:db-path [::spec/events]
           :href    (:id @nuvlabox)})
-       {:menuItem {:content "Deployments"
+       {:menuItem {:content (r/as-element [:span "Deployments"
+                                           [ui/Label {:circular true
+                                                      :size     "mini"
+                                                      :attached "top right"}
+                                            (count (:resources deployments))]])
```

Comment:
we can't use the count of resources, because deployments are paged. You will get max number of elements on page (e.g. 8) even NuvlaEdge has 100 deployments

Reply:
You fixed that already right?

Reply:
yes